lindi/LindiH5pyFile/LindiH5pyFile.py

```diff
@@ -209,6 +308,29 @@ def file(self):
     def name(self):
         return self._the_group.name
 
+    @property
+    def ref(self):
+        return self._the_group.ref
+
+    ##############################
+    # write
+    def create_group(self, name, track_order=None):
+        if self._mode not in ['r+']:
+            raise Exception("Cannot create group in read-only mode")
+        if track_order is not None:
+            raise Exception("track_order is not supported (I don't know what it is)")
```

Comment:
if `track_order` is true when creating an `h5py.Group`, then when iterating through the objects within the group, they are returned in order of creation, otherwise it is in ascending alphanumeric order. PyNWB doesn't use this. But I don't think it would hurt to pass this argument to `self._the_group.create_group(name)` below.